src/MovieDetail/MovieDetail.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './MovieDetail.css';
+
+const MovieDetail = ( {movie, goBack} ) => {
+  return (
+    <div className='movie-detail-wrapper' style={{
+        backgroundImage: 'url('+ movie.backdrop_path +')'
+      }}>
+      <article className='movie-expanded'>
+      <i onClick={() => goBack()} className="fas fa-arrow-left fa-2x"></i>
```

Comment:
We might want to add an ARIA role to indicate that the icon is a button. This icon may also give us some issues if the background being displayed is a light color